src/hist.jl

```diff
@@ -532,4 +533,4 @@ Calculate the midpoints (pairwise mean of consecutive elements).
 """
 midpoints(v::AbstractVector) = [middle(v[i - 1], v[i]) for i in 2:length(v)]
 
-midpoints(r::Range) = r[1:(end - 1)] + step(r) / 2
+midpoints(r::AbstractRange) = broadcast(+, r[1:(end - 1)], step(r) / 2)
```

Comment:
Doesn't `.+` give the same result? Or is there a precedence issue?

More importantly, it looks like a bug on 0.6 makes `broadcast` return a vector rather than a range. The simplest solution is to use `if VERSION`, keeping `+` on 0.6. Femtocleaner will remove it automatically when we require 0.7.

Reply:
Okay.  Any hints on how I would find the minimum version number in the 0.7.0-DEV series on which to base the conditional?  That is, what change should I be looking for?

Reply:
To be safe, you can just take the commit you are using. That way on older 0.7 versions the deprecation warning will be printed but tests will pass.